Filter/Field/TextFilterField.php

```diff
@@ -0,0 +1,42 @@
+<?php
+// https://www.elastic.co/guide/en/elasticsearch/client/php-api/current/_search_operations.html
```

Comment:
If this provides useful reference you should add it to the class docblocks, this isn't really a supported way to provide documentation